src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -161,6 +154,8 @@
   private final Bucket bucket;
   private final String project;
   private final RetryPolicy<Object> commitRetryPolicy;
+  private final Map<TableId, Long> latestSeenSequence;
+  private final Map<TableId, Long> latestMergedSequence;
```

Comment:
Quick question? are these maps thread-safe?

Reply:
they are not, but the methods in this class that access them are synchronized. I can add a comment

Reply:
Cool.